### PR TITLE
fix: icon aligned when event has a short or long name

### LINKED ISSUE #6187: Video Tab: Icons disaligned when event or video room has a long name
In the video tab overview first column the icons are disaligned when an event or video room name has a longer name. Please ensure the icons stay aligned.

Screenshot from 2020-12-31 09-08-26

### PR DESCRIPTION
**Pull request by afzal442**
Fixes #6187 

#### Changes proposed in this pull request:

- Used d-flex, ml-auto to have proper alignment
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by afzal442** [comment]
After the commit changes proposed
c2

File: app/templates/components/ui-table/cell/events/view/videoroom/cell-stream-title.hbs [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<div class="d-flex">
+<div class="ml-auto">
   {{@record}}
-  <div class="ml-auto">
+  <div class="d-flex">
```
Comment (iamareebjamal) [inline app/templates/components/ui-table/cell/events/view/videoroom/cell-stream-title.hbs:3]: This just looks like a swap for no good reason. This is not hit and trial. Every change should have a reason

Comment (afzal442) [reply app/templates/components/ui-table/cell/events/view/videoroom/cell-stream-title.hbs:3]: But I tried out with various way. Sometime I failed with mobile view so this logic became valid for me. Thanks. Any suggestions?

**Comment by afzal442** [comment]
Sir, I am closing this for now. If somebody else has better logic, can create PR there.

**Pull request closed without merging**